lib/utils/deep-assign.js

```
@@ -20,14 +20,14 @@ __export(stdin_exports, {
   deepAssign: () => deepAssign
 });
 module.exports = __toCommonJS(stdin_exports);
-var import_validate = require("./validate");
+var import_basic = require("./basic");
 const { hasOwnProperty } = Object.prototype;
```

Biome (1.9.4)

[error] Do not shadow the global "hasOwnProperty" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)